src/osd/OSD.cc

```diff
@@ -2205,7 +2205,18 @@ bool OSD::asok_command(string admin_command, cmdmap_t& cmdmap, string format,
       pg->unlock();
     }
     f->close_section();
-  } else {
+  } else if (admin_command == "compact") {
+    dout(1) << "triggering manual compaction" << dendl;
+    utime_t start = ceph_clock_now();
+    store->compact();
+    utime_t end = ceph_clock_now();
```

Comment:
might want to use `ceph::coarse_mono_clock` instead, as we are switching to the chrono library for timing.